### PR TITLE
Add services

### PR DESCRIPTION
**Pull request by ThibaultBardinetLanglois**

### CODE REVIEW

**Review by ced37 — DISMISSED** [review]

**Review by ced37 — DISMISSED** [review]

**Review by StephaneSauvaget — DISMISSED** [review]

**Review by floriane-cheneveau — DISMISSED** [review]

File: src/Controller/ProfessionnalController.php [hunk 1]
```diff
@@ -22,4 +22,15 @@ public function list(ProfessionnalRepository $proRepository): Response
         $professionnal = $proRepository->findAll();
         return $this->render('professionnal/list.html.twig', ['professionnels' => $professionnal]);
     }
+
+    /**
+     * @Route("show/{id}", methods={"GET"}, name="show")
+     */
+    public function show(int $id, ProfessionnalRepository $proRepository): Response
+    {
+        $pro = $proRepository->findOneById($id);
```
Comment (RomainClair) [inline src/Controller/ProfessionnalController.php:31]: findOneById() can be replace by just find()

**Review by RomainClair — CHANGES_REQUESTED** [review]
It may not be necessary to have the font licence file and readme in your asset directory.
You should provide different file format for font. Like woff2 for example.
The asset directory is not accessible. You may store your fonts in the public directory or see webpack capabilities for dealing with fonts.

The Version*** (migration) file is not compatible with the actual dev database state.
You must generate a new one that can be apply form the dev state

**Review by StephaneSauvaget — DISMISSED** [review]
Nice job! :)

**Review by ced37 — DISMISSED** [review]

**Review by StephaneSauvaget — DISMISSED** [review]

**Review by ced37 — DISMISSED** [review]

**Review by RomainClair — COMMENTED** [review]
As you use only two font files, you could embed only the needed ones.

**Review by ced37 — APPROVED** [review]

**Review by StephaneSauvaget — DISMISSED** [review]

**Review by ced37 — DISMISSED** [review]

**Review by RomainClair — APPROVED** [review]

**Review by StephaneSauvaget — APPROVED** [review]

**Review by ced37 — APPROVED** [review]

**Pull request merged**